tests/test_fsclient.py

```diff
@@ -42,15 +38,108 @@
 DEFAULT_RC = {}
 rc = DEFAULT_RC
 
+
 # FileSystemClient methods tested here
 def test_is_alive():
-    expected = True  #filesystem is always alive!
+    expected = True  # filesystem is always alive!
     fsc = FileSystemClient(rc)
     actual = fsc.is_alive()
+
     assert actual == expected
 
-@pytest.mark.skip("Not written")
+
 def test_open():
+    fsc = FileSystemClient(rc)
+    fsc.open()
+
+    assert isinstance(fsc.dbs, type(defaultdict(lambda: defaultdict(dict))))
+    assert isinstance(fsc.chained_db, type(dict()))
+    assert not fsc.closed
+
+
+@pytest.mark.skip("Not written")
+def test_load_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_cif():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_database():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_cif():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_database():
+    pass
+
+
+def test_close():
```

Comment:
How about this.  I think it more explicitly tests that close is working because it ensures that something is open before it closes it.  I think it is probably better to pass in rc explicitly too for greater readibility (I should have done that with open I guess):
```
    def test_close():
        fsc = FileSystemClient(rc)
        assert fsc.open
        assert fsc.dbs == rc.databases
        actual = fsc.close()
        assert fsc.dbs is None
        assert fsc.closed
```

Reply:
I couldn't reference rc since I had to make a new class. For now, I just checked if fsc.dbs is an `instanceof` nested dicts.